README.md

```diff
@@ -900,6 +900,79 @@ _You can enable the following settings in Xcode by running [this script](resourc
     …
   }
   ```
+  
+  </details>
+
+* <a id='wrap-multiline-conditional-assignment'></a>(<a href='#wrap-multiline-conditional-assignment'>link</a>) **Add a line break after the assignment operator (`=`) before a multi-line `if` or `switch` expression**, and indent the following `if` / `switch` expression. If the declaration fits on a single line, a line break not required. [![SwiftFormat: wrapMultilineConditionalAssignment](https://img.shields.io/badge/SwiftFormat-wrapMultilineConditionalAssignment-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapMultilineConditionalAssignment)
+
+  <details>
+
+  #### Why?
+  
+  This makes it so that `if` and `switch` expressions always have the same "shape" as standard `if` and `switch` statements, where:
+  1. The `if` / `switch` keyword is always the left-most token on a dedicated line of code
+  2. The conditional branches are always to the right of and below the `if` / `switch` keyword
+
+  This is most consistent with the formatting of `if` / `switch` statements in other contexts, and thus makes it easier to recognize that the code is using an `if` or `switch` expression at a glance. 
```

Comment:
```suggestion
  This is most consistent with how the `if` / `switch` keywords are used for control flow, and thus makes it easier to recognize that the code is using an `if` or `switch` expression at a glance. 
```
I'm not 100% sure on this but I wanted to throw out the idea.

Reply:
Good idea